src/V3Trace.cpp

```diff
@@ -627,6 +651,32 @@ class TraceVisitor final : public VNVisitor {
         }
     }
 
+    DtypeFuncs createNonConstDtypeTraceFunctions(const AstTraceDecl* declp) {
+        AstNodeDType* dtypep = declp->valuep()->dtypep();
+        auto pair = m_dtypeFuncs.emplace(dtypep, DtypeFuncs{});
+        if (pair.second) {
+            FileLine* const flp = declp->fileline();
+            AstCFunc* fullFuncp = newCFunc(VTraceType::FULL, nullptr, 0, 0, declp);
```

Comment:
Const needed lots of places.

```suggestion
            AstCFunc* const fullFuncp = newCFunc(VTraceType::FULL, nullptr, 0, 0, declp);
```